templates/libraries/okhttp-gson/JSON.mustache

```diff
@@ -218,7 +217,7 @@ public class JSON {
                     return null;
                 default:
                     String bytesAsBase64 = in.nextString();
-                    return Base64.decodeBase64(bytesAsBase64);
+                    return Base64.getDecoder().decode(bytesAsBase64);
```

Comment:
```suggestion
                    return Base64.getDecoder().decode(bytesAsBase64);
```
This is actually not the proper mustache template dir. You need the other one :)

Reply:
-> jersey3

Reply:
There is not base64 related code in `templates/libraries/jersey3/`, right ? @jillingk